semgrep/tests/e2e/targets/basic/regex-multiline-mode.txt

```diff
@@ -0,0 +1,8 @@
+hi
+
+# ERROR: test-regex-multiline-mode
```

Comment:
you could have used semgrep-core/tests/OTHER/rules/ for this test instead of doing it via a pytest.